rover/usbreset.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```

Comment:
lets use python2 to be consistent with all of the rest of the code in the repo and avoid potential future confusion

Reply:
Done